packages/aws-cdk-lib/custom-resources/test/provider-framework/provider.test.ts

```diff
@@ -180,6 +181,11 @@ test('if isComplete is specified, the isComplete framework handler is also inclu
   new cr.Provider(stack, 'MyProvider', {
     onEventHandler: handler,
     isCompleteHandler: handler,
+    waiterStateMachineLogOptions: {
+      includeExecutionData: true,
+      level: LogLevel.ALL,
+    },
+    disableWaiterStateMachineLogging: false,
```

Comment:
```suggestion
```
Redundant.
Can you please add a unit test for `disableWaiterStateMachineLogging: true` and modify the `hasResourceProperties('AWS::StepFunctions::StateMachine'` of one of the above to verify that `LoggingConfiguration` is set by default?

Reply:
I changed, please see [them](https://github.com/aws/aws-cdk/pull/27310/commits/3ebccbd1d2cc5a4ed06c89582e618ff1c438547c#diff-f4e9d2a297a5917954bf071f5007abf62375eaff321e3f9380576ade2e5aafb1R265-R301).